tests/integ/__init__.py

```diff
@@ -23,9 +23,10 @@
 TUNING_DEFAULT_TIMEOUT_MINUTES = 20
 TRANSFORM_DEFAULT_TIMEOUT_MINUTES = 20
 PYTHON_VERSION = 'py' + str(sys.version_info.major)
-HOSTING_P2_UNAVAILABLE_REGIONS = ['ca-central-1', 'us-west-1', 'eu-west-2']
-HOSTING_P3_UNAVAILABLE_REGIONS = ['ap-southeast-1', 'ap-southeast-2', 'ap-south-1', 'ca-central-1',
-                                  'eu-west-2', 'us-west-1']
+HOSTING_NO_P2_REGIONS = ['ca-central-1', 'eu-west-2', 'us-west-1']
+HOSTING_SCARCE_P2_REGIONS = HOSTING_NO_P2_REGIONS + ['eu-central-1']
```

Comment:
what's the benefit of separating these?

Reply:
hmm. not much the way i've implemented it. i'll combine them.